CHANGELOG.md

```diff
@@ -87,6 +87,22 @@ where fiber-velocity can be estimated from the state or assumed to be zero if th
   `preserveMassDist` arguments were swapped and the `preserveMassDist` argument
   is no longer optional. The default argument for `preserveMassDist` in OpenSim
   3.3 was `false`. (PR #1994)
+- A GeometryPath without PathPoints is considered invalid, since it does not represent a physical system. You must specify PathPoints to define a valid GeometryPath for a Muscle, Ligament, PathSpring, etc... that is added to a Model. (PR #1948)
```

Comment:
It would be good to keep the lines under 80 columns, to make it easier to read this file in all contexts (some editors may not do soft wraps).